src/engine/tests/drpc_comm_tests.c

```diff
@@ -0,0 +1,177 @@
+/**
+ * (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+/*
+ * Simple dRPC integration tests
+ */
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <string.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <daos_errno.h>
+#include <daos/drpc_types.h>
+#include <daos/drpc.h>
+#include <daos/tests_lib.h>
+#include <daos/test_utils.h>
+#include "drpc_test_listener.h"
+#include <daos/drpc_test.pb-c.h>
+
+#define CHUNK_SIZE (1 << 17) /* dRPC chunk size = 1 MB */
```

Comment:
128MiB instead of 1MB?

Reply:
Oops, should be 1 << 20. This test really should pull this value from drpc_types.h anyway.